assets/src/dashboard/style.css

```diff
@@ -20,9 +20,22 @@
  * Hides all controls that interfere with the dashboard UI.
  */
 
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%; /* root font size is 10px */
+}
+
 body {
   background: #fff;
-  color: #1a1d1f;
+  color: #1a1d1f; /* theme.colors.gray900 */
+  font-family: 'Google Sans', Sans Serif;
```

Comment:
This overrides the default font for all of WordPress admin. ⚠️

Reply:
Ah that's a good point, we can scope it to the `id` for the container of the dashboard.